bin/webindex

```diff
@@ -88,29 +160,45 @@ ui)
     DROPWIZARD_CONFIG=$WI_HOME/conf/dropwizard.yml
     echo "Running with dropwizard config at $DROPWIZARD_CONFIG"
   fi
-  java -jar $WI_HOME/modules/ui/target/webindex-ui-0.0.1-SNAPSHOT.jar \
-    server $DROPWIZARD_CONFIG &> $COMMAND_LOGFILE &
-  echo "Started UI.  Logs are being output to $COMMAND_LOGFILE"
+  COMMAND="java -jar $WI_HOME/modules/ui/target/webindex-ui-0.0.1-SNAPSHOT.jar \
+    server $DROPWIZARD_CONFIG"
+  if [ "$2" != "-fg" ]; then
+    nohup ${COMMAND} &> $COMMAND_LOGFILE &
+    echo "Started UI.  Logs are being output to $COMMAND_LOGFILE"
+  else
+    ${COMMAND}
+  fi
   ;;
 splits)
-  . $BIN_DIR/impl/base.sh "${*:2}"
-  $SPARK_SUBMIT --class io.fluo.webindex.data.CalcSplits \
-    --master yarn-client \
-    --num-executors `get_prop sparkExecutorInstances` \
-    --executor-memory `get_prop sparkExecutorMemory` \
+  . $BIN_DIR/impl/base.sh
+  COMMAND="$SPARK_SUBMIT --class io.fluo.webindex.data.CalcSplits \
+    $COMMON_SPARK_OPTS \
     --conf spark.shuffle.service.enabled=true \
-    $WI_DATA_DEP_JAR $WI_HOME/conf/data.yml &> $COMMAND_LOGFILE &
-  echo "Started splits calculation.  Logs are being output to $COMMAND_LOGFILE"
+    $WI_DATA_DEP_JAR $2"
+  if [ "$2" != "-fg" ]; then
+    nohup ${COMMAND} &> $COMMAND_LOGFILE &
+    echo "Started splits calculation.  Logs are being output to $COMMAND_LOGFILE"
+  else
+    ${COMMAND}
+  fi
   ;;
 *)
   echo -e "Usage: webindex <command> (<argument>)\n"
   echo -e "Possible commands:\n"
-  echo "  copy        Copies CommonCrawl data files from AWS into HDFS init & load directories"
-  echo "  init        Starts Spark job that initializes Fluo & Accumulo tables using data in HDFS init directory "
-  echo "  load        Starts Spark job that adds pages to Fluo using data in the HDFS load directory"
-  echo "  ui          Runs the webindex UI"
-  echo "  reindex     Runs Spark data in Fluo and recreates Accumulo indexes"
-  echo "  splits      Calculate splits"
+  echo "  getpaths <DATE>             Retrieves paths file for specified crawl <DATE> (i.e 2015-18) and stores file in paths/ dir"
+  echo "                              See https://commoncrawl.org/the-data/get-started/ for possible crawl dates"
+  echo "  copy <DATE> <RANGE> <DEST>  Copies CommonCrawl data files from AWS given <DATE> and <RANGE> (i.e 0-8) into HDFS <DEST> directory"
```

Comment:
It's mentioned at the bottom of the help.  I decided to mention it once rather than for each command.